charts/gardener-extension-shoot-lakom-admission/charts/runtime/values.yaml

```diff
@@ -1 +0,0 @@
-../../values.yaml
```

Comment:
`serviceAccountTokenVolumeProjection` and `gardener.virtualCluster.serviceAccount` are unused values. 


PS. For some reason I cannot post comments on the new `values.yaml` file, github just refuses with error like `line must be part of the diff`

Reply:
Thanks, `serviceAccountTokenVolumeProjection` values have been removed.
`gardener.virtualCluster.serviceAccount` is used at several places, e.g. charts/gardener-extension-shoot-lakom-admission/charts/application/templates/rbac.yaml, line 50

Reply:
`gardener.virtualCluster.serviceAccount` is used in the `application` chart, but not in the runtime, that is why I propose to remove it.

Reply:
You're right, thanks!